enf/domain_ratelimit.go

```diff
@@ -0,0 +1,66 @@
+package enf
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+)
+
+// DomainService handles communication with the domain-related methods
+// of the ENF API.
+type DomainService service
+
+// DomainRateLimits represents the values of rate limits for a domain.
+type DomainRateLimits struct {
+	PacketsPerSecond *int `json:"packets_per_second"`
+	PacketsBurstSize *int `json:"packets_burst_size"`
+	BytesPerSecond   *int `json:"bytes_per_second"`
+	BytesBurstSize   *int `json:"bytes_burst_size"`
+}
+
+// domainRateLimitResponse represents the standard API response for
+// all relevant network or endpoint rate limit endpoints.
+type domainRateLimitResponse struct {
+	Data []*DomainRateLimits    `json:"data"`
+	Page map[string]interface{} `json:"page"`
+}
+
+// GetDefaultRateLimits gets the default rate limits for the given domain.
+func (s *DomainService) GetDefaultRateLimits(ctx context.Context, domain string) (*DomainRateLimits, *http.Response, error) {
```

Comment:
Should these method names indicate that these are "endpoint" rate limits? E.g., `GetDefaultEndpointRateLimits`.

Doing so is more accurate. It's not a rate limit for the domain; it's a default rate limit for the endpoints in the domain.